pages/modules.html

```diff
@@ -137,7 +137,7 @@ <h3>Build everything</h3>
                 <div class="zero-clipboard"><span class="btn-clipboard">Copy</span></div>
                 <figure class="highlight">
                     <pre><code class="language-xml" data-lang="xml"><span class="c">&lt;?xml version="1.0" encoding="utf-8" ?&gt;</span>
-<span class="nt">&lt;Project <span class="na">xmlns=</span><span class="s">"http://schemas.microsoft.com/developer/msbuild/2003"</span>&gt;</span>
+<span class="nt">&lt;Project <span class="na">xmlns=</span><span class="s">"https://schemas.microsoft.com/developer/msbuild/2003"</span>&gt;</span>
```

Comment:
I don't think these can change, they are just schema links that aren't actually downloaded from a remote resource.

The schema file has this:

```xml
<xs:schema
  targetNamespace="http://schemas.microsoft.com/developer/msbuild/2003"
  xmlns:xs="http://www.w3.org/2001/XMLSchema"
  xmlns:msb="http://schemas.microsoft.com/developer/msbuild/2003"
  elementFormDefault="qualified">
```

So please revert these and I'll merge.